.github/workflows/lib-scorecard.yaml

```
@@ -0,0 +1,33 @@
+name: "OSSF"
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  analysis:
+    name: Analysis
+    runs-on: ubuntu-latest
```

we have followed a locked version:
```suggestion
    runs-on: ubuntu-22.04
```

(applies to other `ubuntu-latest` too)